skia/BUILD.gn

```diff
@@ -699,6 +700,21 @@ skia_source_set("skia_core_and_effects") {
   visibility = [ ":skia" ]
 }
 
+if (skia_support_skparagraph) {
```

Comment:
Is this check required? 
skparagraph will be required as default, right?

Reply:
Since skparagraph is enabled by default, removed the check.